cmd/contour/serve.go

```diff
@@ -231,7 +243,7 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 
 	// Before we can build the event handler, we need to initialize the converter we'll
 	// use to convert from Unstructured.
-	converter, err := k8s.NewUnstructuredConverter()
+	converter, err := k8s.NewUnstructuredConverter(mgr.GetScheme())
```

Comment:
Does it make more sense to create the scheme with the gvk's that we care about and pass the scheme in as a `manager.Option`?

Reply:
Yeah that's sort of what was happening here: https://github.com/projectcontour/contour/blob/4885db6354ec9cbb92ef183e2c02e71c9aee9cbd/internal/k8s/scheme.go#L27